### PR TITLE
Improve doc on Jupyter tools

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: docs/ROADMAP.md [hunk 1]
```diff
@@ -65,7 +65,11 @@ and Tekton Pipelines.
...
-similar functionality as thamos, but embeds the dependencies and the locked dependencies within the meta information
-of the Jupyter Notebook file itself.
+Jupyter tools focused on the Data Scientist workflow have been created to help them in dependencies management:
+
+* Jupyter Notebook extension [nb-requirements](https://github.com/thoth-station/jupyter-nbrequirements)
+
+* JupyterLab extension [jupyterlab-requirements](https://github.com/thoth-station/jupyterlab-requirements)
+
+They feature similar functionality as thamos, but embedding the dependencies and the locked dependencies within the meta information
+of the Jupyter Notebook file itself togheter with creating dependenices files in the requested requirement formats.
```
Comment (bissenbay) [inline docs/ROADMAP.md:75]:
```suggestion
of the Jupyter Notebook file itself together with creating dependencies files in the requested requirement formats.
```

Comment (pacospace) [reply docs/ROADMAP.md]: Thanks @bissenbay!!

File: docs/ROADMAP.md [hunk 2]
```diff
@@ -65,7 +65,11 @@ and Tekton Pipelines.
 
 ### Jupyter Requirements Management
 
-A tool focused on the Data Scientist workflow is [nb-requirements](https://github.com/thoth-station/jupyter-nbrequirements)
-and [jupyterlab-requirements](https://github.com/thoth-station/jupyterlab-requirements), it features a
-similar functionality as thamos, but embeds the dependencies and the locked dependencies within the meta information
-of the Jupyter Notebook file itself.
+Jupyter tools focused on the Data Scientist workflow have been created to help them in dependencies management:
+
+* Jupyter Notebook extension [nb-requirements](https://github.com/thoth-station/jupyter-nbrequirements)
```
Comment (fridex) [inline docs/ROADMAP.md:70]:
```suggestion
* Jupyter Notebook extension [jupyter-nbrequirements](https://github.com/thoth-station/jupyter-nbrequirements)
```

**Review by fridex — COMMENTED** [review]
one minor comment, otherwise lgtrm

File: docs/ROADMAP.md (hunk 2 above)
Comment (pacospace) [reply docs/ROADMAP.md]: Thanks!!

File: docs/ROADMAP.md [hunk 3]
```diff
@@ -65,7 +65,11 @@ and Tekton Pipelines.
 
 ### Jupyter Requirements Management
 
-A tool focused on the Data Scientist workflow is [nb-requirements](https://github.com/thoth-station/jupyter-nbrequirements)
-and [jupyterlab-requirements](https://github.com/thoth-station/jupyterlab-requirements), it features a
-similar functionality as thamos, but embeds the dependencies and the locked dependencies within the meta information
-of the Jupyter Notebook file itself.
+Jupyter tools focused on the Data Scientist workflow have been created to help them in dependencies management:
```
Comment (goern) [inline docs/ROADMAP.md:68]: I think its "help with" not "help in" @harshad16 ?

**Review by goern — CHANGES_REQUESTED** [review]

File: docs/ROADMAP.md (hunk 3 above)
Comment (harshad16) [reply docs/ROADMAP.md:68]: help with seems more suitable.

**Pull request merged**